app/models/profile.rb

```diff
@@ -1,3 +1,8 @@
 class Profile < ApplicationRecord
+  validates :job, format: {with: /\A[a-zA-Z0-9 ]+\z/}, allow_nil: true
+  validates :linkedin, format: { with: /.*linkedin.\S*\/\S*/ }, allow_nil: true
+  validates :github, format: { with: /.*github.\S*\/\S*/ }, allow_nil: true
```

Comment:
fancy! I assume we allow nil initially and then they fill in? I assume profile is created on user creation? I guess this gets at what I was asking earlier (re value of profile) but not relevant to this pr